redesign/css/dwyl.css

```diff
@@ -60,14 +60,22 @@ body {
   font-size: 4rem;
 }
 
-.bw-s {
-  border-width: thin;
-}
-
 .mw-21 {
   max-width: 21rem;
 }
 
+/*Navbar*/
+
+.menu {
+  min-height: 4rem;
+  max-height: 0;
+  transition: max-height 1s;
```

Comment:
The transition feels a bit slow, could we make it a bit faster?

Reply:
```suggestion
transition: max-height 0.5s;
```
😉

Reply:
Made it faster for you @SimonLab https://github.com/dwyl/learn-tachyons/pull/52/files#diff-910fef80905cec0740d8c9b257c89defR47 😉